website/plugins/gatsby-transformer-react-docgen-typescript/gatsby-node.js

```diff
@@ -0,0 +1,4 @@
+"use strict";
+
+exports.onCreateNode = require(`./on-node-create`).default;
+exports.setFieldsOnGraphQLNodeType = require(`./extend-node-type`).default;
```

Comment:
I wonder if you can reuse the files that you copied without modifications

Reply:
still don't get it. reusing from where? 
You told me to use require,
The files copied without modification are needed in the `getsby-node.js` of the plugin. For example, `parse.js`. They are nowhere else (e.g. not in node_modules, etc). From where do I use require?